ledger/ledger-runner-common/src/test/scala/com/daml/ledger/runner/common/PureConfigReaderWriterSpec.scala

```diff
@@ -751,6 +751,7 @@ class PureConfigReaderWriterSpec
   val validIndexServiceConfigValue =
     """|
       |acs-streams {
+      |    contract-processing-parallelism=8
```

Comment:
so we are already in `acs-streams` scope so `contract` doesn't bring too much extra imho. So I would either use the extra verbosity to add some entropy (e.g. `per-stream-processing-parallelism`) or just simplify like `processing-parallelism`. wdyt?

Reply:
Config keys here describe processing of event ids (event seq ids) and event payloads (for acs: contracts).
I used to `contract` to differentiate from ids.